doc/shared/Install.rst

```diff
@@ -1142,6 +1161,27 @@ addressing specific configurations when using the supported third
 party libraries.
 
 
+.. _Installation.CMake.ExternalLibraries.Ginkgo:
+
+Building with Ginkgo
```

Comment:
This section should include any prohibited configurations for building with Ginkgo.  For example, elsewhere it looks like Ginkgo is incompatible with extended precision, and that when using at least the CUDA backend that it must use 32-bit `sunindextype`.  Does it have others as well (e.g., 32-bit `sunindextype` for HIP or SYCL)?

Reply:
I don't think there are any restrictions like this due to our interfaces. Ginkgo internally has some issues with 64-bit, but they may resolve that in a future release, so I don't think we should place a constraint on it.

Reply:
So even extended precision for `sunrealtype` should work?

Reply:
Yes, there is no reason our interfaces should prevent it. We just pass `sunrealtype` as a template parameter to Ginkgo. They may or may-not support it in depending on the algorithm etc, but it would be a tall order for us to determine which ones will work or not and maintain constraints for them all.

Reply:
@balos1 When I ran across the CUDA+int64 error I added a CMake error in `SundialsGinkgo.cmake` that probably needs to be moved inside a `if(NOT GINKGO_WORKS)` block so users than bypass if support gets added

Reply:
I think we could probably just remove that check altogether. Wasn't it specific to a matrix format?

Reply:
Yes, I it was specific to the CSR format

Reply:
If we don't exclude it a configure time, we'll need to exclude the `cv_heat2D_ginkgo` example from the CI when +cuda+int64

Reply:
Yes, I think we should just exclude that example instead of adding a configure time constraint.